frontend/src/lib/WebsocketConnection.tsx

```diff
@@ -517,12 +522,22 @@ export function doHealthPing(
     connect()
   }
 
-  // Make sure we don't retry faster than timeoutMs. This is required because
-  // in some cases things fail very quickly, and all our fast retries end up
-  // bogging down the browser.
+  // Make sure we don't retry faster than minimumTimeoutMs. This is required
+  // because in some cases things fail very quickly, and all our fast retries
+  // end up bogging down the browser.
   const retry = (errorNode: React.ReactNode): void => {
     const tryDuration = (Date.now() - tryTimestamp) / 1000
```

Comment:
It looks like `tryDuration` is in seconds as opposed to milliseconds - is this intentional?

Reply:
No idea, that part was already in the code. It looks like a bug. That part of the function doesn't even feel very useful, since it only serves to make the ping interval not slowed down by the time it takes to do the ping, which is usually very low. I could fix it, to try and get minor responsiveness benefits, or I could remove it, because it currently isn't doing much of anything and nobody noticed, and have the function be a little simpler. I'm leaning towards removal, but I'm curious what you think

Reply:
i think it's fine to remove it!